readthedocs/settings/docker_compose.py

```diff
@@ -45,6 +45,16 @@ class DockerBaseSettings(CommunityDevSettings):
     ADSERVER_API_KEY = None
     ADSERVER_API_TIMEOUT = 2  # seconds - Docker for Mac is very slow
 
+    # New templates
+    @property
+    def RTD_EXT_THEME_DEV_SERVER_ENABLED(self):
+        return os.environ.get('RTD_EXT_THEME_DEV_SERVER_ENABLED') is not None
+
+    @property
+    def RTD_EXT_THEME_DEV_SERVER(self):
+        if self.RTD_EXT_THEME_DEV_SERVER_ENABLED:
+            return "http://assets.community.dev.readthedocs.io:10001"
```

Comment:
Any reason we are exposing this? We could just do `webpack:10001`

Reply:
It will depend how this setting is used. If the URL is accessed from another container `webpack:10001` will work, but if it's accessed from the host, it won't because it needs to resolve to `localhost:10001` in the end.

@agjohnson in which part of the code you make usage of this variable?

Reply:
The client directly hits this URL, it's not accessed from another container.